### PR TITLE
fix: close form results in undesired behaviour

### LINKED ISSUE #52: Closing a form functionality not working appropriately
### Describe the bug
The form is visible as open and closed in the same display. 

### Screenshot
Screenshot from 2020-08-15 23-15-38

### Expected behavior
When the admin closes the form , the status of the form should be updated appropriately. currently the same form is displayed two times on the screen. Only one instance of the form should be displayed.

### PR DESCRIPTION
**Pull request by bismitaguha**
### Description

Fixes #52 

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

Manual testing
Screenshot from 2020-08-27 15-21-55

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: q [hunk 1]
```diff
@@ -0,0 +1,131 @@
+[33mcommit 0e2a36e664ec5c66e32ecf5a80ff153e6eef9b04[m[33m ([m[1;36mHEAD -> [m[1;32mview-form[m[33m)[m
```
Comment (sidvenu) [inline q]: Should be removed?

**Review by sidvenu — CHANGES_REQUESTED** [review]
@bismitaguha is this PR complete? Can you attach a GIF?

**Review by sidvenu — DISMISSED** [review]
LGTM. @bismitaguha can you attach a GIF here?

**Comment by bismitaguha** [comment]
@sidvenu @abha224 I showed this in the demo. Please merge this PR.

**Comment by bismitaguha** [comment]
@abha224 Modified the errors!!!
Screenshot from 2020-08-27 15-21-55

File: src/components/PublishedForm.js [hunk 2]
```diff
@@ -129,6 +129,7 @@ class PublishedForm extends Component {
     render() {
         const { publishedform, type } = this.props
         const { deleted } = this.state
+        console.log(this.state)
```
Comment (sidvenu) [inline src/components/PublishedForm.js]: To be removed

**Review by sidvenu — CHANGES_REQUESTED** [review]

**Review by sidvenu — APPROVED** [review]

**Review by abha224 — APPROVED** [review]
Reviewed and tested. :+1:

**Pull request merged**